com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/MyTextDocumentServiceTest.java

```diff
@@ -245,7 +245,7 @@ public void testCodeActionsEndpoint() {
     when(actions.collect(params)).thenReturn(expected);
 
     MyTextDocumentService service =
-        new MyTextDocumentService(null, null, null, null, null, broker, actions);
+        new MyTextDocumentService(null, null, null, null, null, broker, actions, null);
```

Comment:
Better have another minimalist constructor , to get rid-off these nulls

Reply:
I agree I'm so tired to change our tests on each service change.